Automattic/jetpack
# My Jetpack: Simplify Products class

This PR refactors and simplifies the Products class.

* No more hard coded slugs. Slugs only live inside each individual Product class
* No need to load all products information (and make requests to WPCOM) when you only want information about one product (`get_product()`)
* Better method naming (get_products_slugs instead of get_products_names)
* Removed unnecessary methods
* Only one place to change if a new product is added (`get_products_classes`)

#### Changes proposed in this Pull Request:

* Simplify Products class

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Tests should pass
* Everything should work as before... browse a little, activate, deactivate... Nothing should change
